pkg/remoteresolution/resolver/git/testdata/test-org/test-repo/refs/other/pipelines/example-pipeline.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: tekton.dev/v1beta1
```

Comment:
NIT: it would be nice to use `v1` instead (the original test data uses `v1beta1` because there was no v1 at the time